src/docs/windows/pwa-api-documentation/pwa-api-documentation.md

```diff
@@ -0,0 +1,312 @@
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    To ensure your PWA is installable, make sure you meet the requirements <a href="https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Installable_PWAs" target="_blank" rel="noopener">listed here</a>, <a href="https://www.pwabuilder.com" target="_blank" rel="noopener">PWABuilder</a> can help you meet these requirements if your PWA does not already.
+  </p>
+</div>
+
+### The Microsoft Store
+
+PWAs can also be installed from the Microsoft Store just like any other app. In fact, the [new Microsoft Store](https://blogs.windows.com/windowsexperience/2021/06/24/building-a-new-open-microsoft-store-on-windows-11/) and updated policies make it easier than ever to get your PWA into the store, even if you use your own payment systems! For more developer information on how this works, check out [our docs](https://blog.pwabuilder.com/posts/bringing-chromium-edge-pwas-to-the-microsoft-store/).
+
+<img src="/docs/windows/pwa-api-documentation/in-store.png" width="350px" alt="What PWA installation from the Microsoft Store looks like" /> 
+
+<div class="how-to">
+  <strong>How To</strong>
+
+  <p>
+    <a href="https://www.pwabuilder.com" target="_blank" rel="noopener">PWABuilder</a> can automatically package and submit Chromium Based Edge PWAs to the store, ensuring that your PWA works the same whether its installed from the Store or Edge itself.
+  </p>
+</div>
+
+## App Experience on Windows
+Once your PWA has been installed by the user (either through the Store or Edge) Windows will treat this PWA just like any other app on Windows. Your PWA can be pinned to the Taskbar and Start Menu:
+
+<img src="/docs/windows/pwa-api-documentation/taskbar.png" width="350px" alt="A screenshot of a PWA in the taskbar" /> 
+
+Your PWA will also open in its own Window:
+
+<img src="/docs/windows/pwa-api-documentation/in-window.png" width="350px" alt="A screenshot of the PWA running in its own window" /> 
+
+And finally, it will also show up in your App Settings:
+
+<img src="/docs/windows/pwa-api-documentation/app-settings.png" width="350px" alt="A screenshot of the PWA in the settings app" /> 
+
+## File Handling and Integrating with the File Explorer
+
+Handling files has always been a challenge for Web Apps as they have never had the same file handling capability as native apps. For example, you could not simply open a file from the File System, download a file or securely write to a file on the users device. With the [File System API](https://web.dev/file-system-access/), your PWA can now use files with the normal experience users expect from apps.
+
+Your PWA can prompt the user to open a file, open mutiple files, open a directory and save files! Your PWA can also prompt the user for write access to an existing file, allowing your pwa to transparently write to a file without any extra downloads needed.
```

Comment:
I'd add a sentence about the file system API opening up new classes of apps, such as text editors, code editors, image editors. Nearly every native app accesses the file system, now web apps can too, and do so in a safe, sandboxed way.